lib/routes.js

```diff
@@ -3,7 +3,7 @@ const axios = require('axios');
 module.exports = function (self, options) {
   self.addRoutes = function() {
     self.route('post', 'page-scan-modal', async (req, res) => {
-      if (!self.apos.permissions.can(req, 'admin')) {
+      if (!self.apos.permissions.can(req, 'admin-seo')) {
```

Comment:
Let's rename this to "admin-seo-scan" to complete making it clear what it lets you do.

Reply:
I've renamed it "admin-seo-page-scan" to match the name of the feature.
Please let me know if it's ok.